src/plugins/cosmos/components/modals/Staking/StakingCommon.ts

```diff
@@ -87,3 +87,13 @@ export const claimSteps = [
   { step: 0, path: ClaimPath.Confirm, label: 'Confirm' },
   { step: 1, path: ClaimPath.Broadcast, label: 'Broadcast' },
 ]
+
+// TODO(gomes): Make this dynamic, this should come from chain-adapters when ready there
+// UnbondingTime is an IBC-chain specific parameter
+// Hardcoded to CosmosHub and Osmosis staking for now, but in the future we should find a better home for this with the right unbonding days per protocol-level unbonding days
+// https://docs.cosmos.network/v0.44/modules/staking/08_params.html
+export const COSMOS_UNBONDING_DAYS = '21'
+export const OSMOSIS_UNBONDING_DAYS = '14'
+
+export const isCosmosStaking = (chainId: ChainId) => chainId === cosmosChainId
+export const isOsmosisStaking = (chainId: ChainId) => chainId === osmosisChainId
```

Comment:
Rename to `isCosmosChainId` / `isOsmosisChainId` to decouple it from the staking terminology

Reply:
Done in b58dfd38099c90bb4fd1d1928d673da1e0fa96f2